tests/test_implementations/test_aggregator.py

```diff
@@ -49,11 +50,13 @@ def test_relationships(self):
         self.assertCountEqual(rels[IS_A], ["UBERON:0010000"])
 
     def test_all_terms(self):
+        """Test all terms."""
         curies = list(self.oi.all_entity_curies())
         self.assertIn(NUCLEUS, curies)
         self.assertIn(INTERNEURON, curies)
 
     def test_relations(self):
+        """Test relations."""
```

Comment:
remember that this isn't just about passing flake8, but it's now time to write descriptions of what and **why** each of these functions, classes, and modules exist.

Reply:
obviously there's a massive technical debt accumulated already where the choices in why code was written is not documented, so this will not be easy.

Reply:
Feel free to jump in if you'd like. As you mention the debt is indeed great and help is always appreciated.